scripts/heat/devstack-heat-template.yaml

```diff
@@ -0,0 +1,164 @@
+heat_template_version: 2016-10-14
+
+description: |
+  devstack on Leap 15.0
+
+parameters:
+  key_name:
+    type: string
+    label: Key-pair name
+    description: Name of key-pair to be used for compute instance
+
+  floating_network_name:
+    type: string
+    label: Floating network name
+    description: Name of key-pair to be used for compute instance
+    default: floating
+
+  flavor_name:
+    type: string
+    label: Flavor
+    description: Flavor of the server
+    default: m1.large
+
+  image_name:
+    type: string
+    label: Image
+    description: Image to boot the server with
+    default: openSUSE-Leap-15.0
```

Comment:
that isn't a public image on the engcloud.  better choices are "openSUSE-Leap-15.0-JeOS.x86_64" or "openSUSE-Leap-15.0-OpenStack.x86_64" or "cleanvm-jeos-SLE12SP4" depending on which bugs you want.

Reply:
Ah, good catch.  However I just tried both of those and they both have broken DNS - I guess it's due to a near empty `resolv.conf` :-/  I don't know why it's empty but probably related to cloud-init:

```
; Created by cloud-init on instance boot automatically, do not edit.                                                                                                     
;
```

Reply:
Interestingly, `/etc/cloud/templates/resolv.conf.tmpl` is identical on both the broken and working images.

Reply:
And /etc/sysconfig/network/config is identical ...